test/e2e/kubectl/kubectl.go

```diff
@@ -1832,6 +1842,20 @@ metadata:
 	})
 })
 
+func findOcc(list []string, item string) []int {
```

Comment:
Why findOcc ? I think in golang we don't have limitations in chars for func names. I would go with a full name for easy understanding. findOccurrences()? Today we understand what's occ in 3 months ahead would be difficult without digging the code.